pom.xml

```
@@ -51,8 +51,7 @@
         <revision>1</revision>
         <changelist>999999-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/trilead-api-plugin</gitHubRepo>
-        <jenkins.version>2.319</jenkins.version>
-        <java.level>8</java.level>
+        <jenkins.version>2.346.2</jenkins.version>
```

```suggestion
        <jenkins.version>2.357</jenkins.version>
```

---

I am waiting for the LTS version

---

Why? Who cares? Lots of plugins are already depending on weeklies.

---

Should have waited for the LTS before merging this then.

---

I'm not particularly eager to set the minimum required version to an unstable version, and I instead prefer to use LTS versions.

---

What about the [proposed baseline](https://groups.google.com/g/jenkinsci-dev/c/JlbiG5pO5l8/m/8gSDmGajAQAJ)?
```suggestion
        <jenkins.version>2.361</jenkins.version>
```
Anyway why would the “stability” of a weekly matter for purposes of setting a minimal version requirement? If CI passes, that is all you should care about. Users should know that actually running an outdated weekly is not “supported”.

---

At any rate, you have three choices:
* Update the baseline to a weekly 2.357 or newer.
* Revert the parent POM update.
* Somehow arrange to override the Java level in the newer parent POMs.

---

Given how many watchers and voters JENKINS-69229 already has, just doing that might have spared a lot of folks some pain. Seems like a small cosmetic issue in comparison.

---

> three choices

IIUC, per https://github.com/jenkinsci/trilead-ssh2/pull/107#issuecomment-1205336436 there's not going to be a new release before this PR gets amended to depend on 2.36x.y. So that's the fourth choice, not doing new releases.

---

> I'm not particularly eager to set the minimum required version to an unstable version, and I instead prefer to use LTS versions.

but what you have on master is unusable anyway...
releasing this today means that users on a weekly will be able to install it, and users will see the upgrade as soon as they upgrade to the LTS (there is no waiting for the LTS to be released, a change of the version here, followed by a release).

At any rate there is nothing preventing you from releasing this today and then releasing an update that depends on the LTS when it is available in 3 weeks time.

---

what is in the main brach in not a release so it could be unstable, is the definition of main branch, a version on develop.